generator/templates/swagger_json_embed.gotmpl

```diff
@@ -1,5 +1,9 @@
 // Code generated by go-swagger; DO NOT EDIT.
 
+
+{{ if .Copyright }}/*{{"\n"}}{{.Copyright}}*/{{ end }}
```

Comment:
I am not sure if comment action works. it is giving me this error when I add comment action

`2017/08/23 11:19:22 Failed to load template serverMain: template: serverMain:127: unexpected EOF`